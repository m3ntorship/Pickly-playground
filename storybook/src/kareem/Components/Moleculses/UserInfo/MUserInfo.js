import React from "react";
import Avatar from "../../Atoms/Avatars/Avatar";
import Info from "../../Atoms/Avatars/UserInfo";

function MUserInfo({ status = "normal", size = "medieum", Name, Hours }) {
  return (
    <div className="flex items-center justify-start h-10">
      <Avatar size={size} status={status} />
      <Info Name={Name} Hours={Hours} />
    </div>
  );
}

export default MUserInfo;
