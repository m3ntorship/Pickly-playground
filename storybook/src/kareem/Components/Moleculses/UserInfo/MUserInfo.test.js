import React from "react";
import renderer from "react-test-renderer";
import MUserInfo from "./MUserInfo";
//status = "normal", size = "medieum", Name, Hours
describe("Render the UserInfo Moleculses ", () => {
  it("Kareem: Render UserInfo with anonymous in large size  ", () => {
    const tree = renderer
      .create(
        <MUserInfo
          status="anonymous"
          size="large"
          Name="Kareem Mohamed"
          Hours="4"
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Kareem: Render UserInfowith default cases Normal case and size medium ", () => {
    const tree = renderer
      .create(<MUserInfo Name="Kareem Mohamed" Hours="4" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
