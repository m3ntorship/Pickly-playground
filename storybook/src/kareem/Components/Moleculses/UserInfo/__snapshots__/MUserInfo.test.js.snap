// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the UserInfo Moleculses  Kareem: Render UserInfo with anonymous in large size   1`] = `
<div
  className="flex items-center justify-start h-10"
>
  <div>
    <img
      alt="./anonymus.png"
      className="w-14 h-14 rounded-xl"
      src="./anonymus.png"
    />
  </div>
  <div
    className="flex-col ml-4"
  >
    <span
      className="h-5 w-48 font-sans text-sm text-black font-bold block"
    >
      Kareem Mohamed
    </span>
    <span
      className="h-5 w-28 font-sans text-sm text-dark-grey block"
    >
      4
       hours ago
    </span>
  </div>
</div>
`;

exports[`Render the UserInfo Moleculses  Kareem: Render UserInfowith default cases Normal case and size medium  1`] = `
<div
  className="flex items-center justify-start h-10"
>
  <div>
    <img
      alt="./avatar.png"
      className="w-10 h-10 rounded-xl"
      src="./avatar.png"
    />
  </div>
  <div
    className="flex-col ml-4"
  >
    <span
      className="h-5 w-48 font-sans text-sm text-black font-bold block"
    >
      Kareem Mohamed
    </span>
    <span
      className="h-5 w-28 font-sans text-sm text-dark-grey block"
    >
      4
       hours ago
    </span>
  </div>
</div>
`;
